pages/common/xml-canonic.md

```diff
@@ -0,0 +1,20 @@
+# xml-canonic
```

Comment:
The title should be identical to how the command is called.
```suggestion
# xml canonic
```

Reply:
Corrected all places. Please make this clear in the guidelines. It can only be inferred by example.